stdlib/_typeshed/__init__.pyi

```diff
@@ -170,6 +170,9 @@ FileDescriptorLike = Union[int, HasFileno]  # stable
 class SupportsRead(Protocol[_T_co]):
     def read(self, __length: int = ...) -> _T_co: ...
 
+class SupportsReadClose(SupportsRead[_T_co]):
+    def close(self) -> None: ...
+
```

Comment:
If this is only used in one file, it would be better to define it in `xml.sax` rather than adding it to `_typeshed` -- you can get stubtest to stop complaining about it not being there at runtime by calling it `_SupportsReadClose` instead of `SupportsReadClose` (stubtest knows that names starting with a single underscore aren't necessarily meant to exist at runtime)

Reply:
Thank you, I've pushed a separated commit to do so!